q2_types/per_sample_sequences/_transformer.py

```diff
@@ -342,3 +344,32 @@ def _9(fmt: PairedEndFastqManifestPhred64) \
     warnings.warn(_phred64_warning)
     return _fastq_manifest_helper(fmt, _write_phred64_to_phred33,
                                   single_end=False)
+
+
+@plugin.register_transformer
+def _12(dirfmt: SingleLanePerSampleSingleEndFastqDirFmt) \
+        -> QIIME1DemuxDirFmt:
+    with dirfmt.manifest.view(FastqManifestFormat).open() as fh:
+        input_manifest = _parse_and_validate_manifest(fh, single_end=True,
+                                                      absolute=False)
+
+    result = QIIME1DemuxDirFmt()
+    fp = os.path.join(str(result), 'seqs.fna')
```

Comment:
No need to change this, but you can also use the operator overloading support with this object:

```python
fp = result / 'seqs.fna'
```

It will handle sticking in the right path separator. I wouldh't worry about making that change, just more of a neat "TIL" type of thing.